roles/openshift_node_problem_detector/tasks/install.yaml

```diff
@@ -7,22 +7,22 @@
 - name: create Node Problem Detector service account
   oc_serviceaccount:
     name: "{{ openshift_node_problem_detector_service_account }}"
-    namespace: "{{ openshift_node_problem_detector_namespace }}"
+    namespace: openshift-node-problem-detector
```

Comment:
Does the ns exist?

Reply:
funny, my testing just ran into that. I was testing in parallel with opening the PR since I thought it was a simple one, but I forgot about that. I'll update the PR momentarily with the create/delete of the new ns